src/user.rs

```diff
@@ -0,0 +1,180 @@
+use crate::models::PassageUser;
+use crate::openapi::apis::configuration::Configuration;
+use crate::openapi::apis::{user_devices_api, users_api};
+use crate::Error;
+
+pub struct User {
+    pub(crate) configuration: Configuration,
```

Comment:
This field's visibility will need to be updated to match the similar field in the `Auth` struct.

Reply:
[ac297a2](https://github.com/passageidentity/passage-flex-rust/pull/30/commits/ac297a257a3e1abdf3c901aa08c8528d23c478fd)